pokecli.py

```diff
@@ -182,10 +182,11 @@ def init_config():
     add_config(
         parser,
         load,
-        long_flag="--spin_forts",
+        long_flag="--spin",
```

Comment:
just go for long_flag="--forts_spin" for now.

rewriting all cli arguments management is a little out of this PR scope no ?